### PR TITLE
Node removing

### PR DESCRIPTION
**Pull request by svalasovich**

### CODE REVIEW

File: contract/pkg/bucket/messages.go [hunk 1]
```diff
@@ -29,6 +29,13 @@ const (
 	OFFLINE
 )
 
+var NodeTags = map[string]byte{
```
Comment (CereVGello) [inline contract/pkg/bucket/messages.go:32]: Tag bring misleading that one node can have two of them in parallel. 
What if we use State?

File: core/pkg/topology/topology.go [hunk 2]
```diff
@@ -97,10 +97,9 @@ func (r *ring) Partitions(nodeId uint32) []Partition {
 		}
 
 		for j := uint(0); j < r.replicationFactor; j++ {
-			vNode := r.vNodes[i]
-			from := vNode.Token()
+			vNodes := r.Replicas(r.vNodes[i].Token())
```
Comment (CereVGello) [inline core/pkg/topology/topology.go:100]: potential out of range.

Comment (svalasovich) [reply core/pkg/topology/topology.go:116]: It's ok, `i` can't go out of range

File: contract/pkg/bucket/messages.go (hunk 1 above)
Comment (svalasovich) [reply contract/pkg/bucket/messages.go:32]: Updated

**Review by pruxa — APPROVED** [review]

**Pull request merged**